pages/en/network/developing.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Developing
+---
+
+Developers are a core part of The Graph. They are responsible for both building and querying subgraphs, unlocking data for the web3 ecosystem.
+
+## Building Subgraphs
+
+Developers that build subgraphs use the [GraphQL Schema Language](https://graphql.org/learn/schema-language/) to define the data they want to make available.
```

Comment:
I think it's worth providing slightly higher level context on how developers use The Graph:
- building and deploying their own subgraphs, so that they can consume them in their protocols or applications
- consuming useful subgraphs built by other developers
The subgraph API makes it easy for developers to build and iterate on their APIs, and the autogenerated GraphQL schema provides a rich and powerful means to access their indexed data.

Reply:
Agreed! Working on that now